data/org.freedesktop.portal.Notification.xml

```diff
@@ -428,10 +428,14 @@
         ActionInvoked:
         @id: the application-provided ID for the notification
         @action: the name of the action
-        @parameter: array which will contain the target parameter for the action, if one was specified
+        @parameter: an array containing additional information
 
         Send to the application when a non-exported action is
         activated.
+
+        The @parameter contains the target parameter for the action, if one was specified,
+        and the `platform-data` as vardict containing an ``activation-token`` (``s``) for `XDG Activation
+        <https://gitlab.freedesktop.org/wayland/wayland-protocols/-/blob/main/staging/xdg-activation/xdg-activation-v1.xml>`_.
```

Comment:
It's not clear to me what the GVariant format for this is after reading it.

Reply:
yes i agree, this is really messy. It's a workaround for https://gitlab.gnome.org/GNOME/glib/-/issues/3333 

I think i want to drop the entire action property and fix it in glib/gio.

Reply:
I removed the `actions` property and moved the purposes to `buttons`. I think this makes much more sense. Thanks for complaining about this :)